cloverage/src/cloverage/report/console.clj

```diff
@@ -0,0 +1,105 @@
+(ns cloverage.report.console
+  (:gen-class)
+  (:require
+   [clojure.string :as cs]
+   [cloverage.report :refer [file-stats total-stats]]))
+
+(def ansi
+  (let [esc \u001b
+        reset "[0m"
+        tbl {:red "[31m"
+             :yellow "[33m"
+             :green "[32m"
+             :bright "[1m"}]
+    (fn [code text]
+      (when text
+        (str esc (get tbl code reset) text esc reset)))))
+
+(defn strip-ansi [text]
+  (when text
+    (cs/replace text #"\u001b\[\d+m" "")))
```

Comment:
p.s. print-table was mostly ripped off from `clojure.pprint/print-table` but made to work with ANSI escape codes